src/rustllvm/RustWrapper.cpp

```diff
@@ -646,14 +646,32 @@ extern "C" LLVMRustMetadataRef LLVMRustDIBuilderCreateStaticVariable(
     bool isLocalToUnit,
     LLVMValueRef Val,
     LLVMRustMetadataRef Decl = NULL) {
+    Constant *InitVal = cast<Constant>(unwrap(Val));
+
+#if LLVM_VERSION_GE(4, 0)
+   llvm::DIExpression *InitExpr = nullptr;
```

Comment:
It looks like these are 3 spaces instead of 4?

FYI, this will have some conflicts with #38317.